contribute/en/Administration.md

```diff
@@ -121,7 +121,7 @@ When a new Issue or Pull Request comes in, do the following:
 We use the Discord/Matrix chat for quicker, loose interaction.
 All decision and results should still be documented in permanent Github Issues, Discussions or Comments.
 
-## Decision making
+## Decision-making
```

Comment:
No hyphen.

Reply:
There are several views on that but I was following the usage [here](https://dictionary.cambridge.org/dictionary/english/decision-making) [here](https://www.collinsdictionary.com/dictionary/english/decision-making) and [here](https://www.merriam-webster.com/dictionary/decision-making), where the noun form uses a hyphen.

Reply:
Adjectival I'd agree with (_decision-making process_; the only example given in Collins, which is poor...) but straight noun form.  Cambridge seems much more set on the hyphen...  Apparently it's because I'm old fashioned.  Which I guess is now "old-fashioned"...